test/models/item_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class ItemTest < ActiveSupport::TestCase
+  should belong_to :itemable
+
+  test 'a item must have a name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.